rules/no-await-in-loop.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @fileoverview Rule to disallow uses of await inside of loops.
+ * @author Nat Mote
+ */
+"use strict";
+
+// Node types which are considered loops.
+var loopTypes = new Set([
```

Comment:
Can we actually use `Set` here since we don't know what version of node people are using?